src/components/space/sync/SyncAdvancedSwitch.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
```

Comment:
In this file you can keep the same structure, but just check it if `mode === ADVANCED_SYNC_MODE`